ocfweb/component/markdown.py

```diff
@@ -118,7 +119,7 @@ def header(self, text, level, raw=None):
         else:
             id = 'h{level}_{title}'.format(
                 level=level,
-                title=re.sub('[^a-z0-9\-_ ]', '', text.lower()).strip().replace(' ', '-'),
+                title=re.sub('[^a-z0-9\-_ ]', '', strip_tags(text).lower()).strip().replace(' ', '-'),
```

Comment:
we want to strip tags here as well, else we get gross ids like:
`#h4_a-hrefdocsserviceswebweb-hostinga-how-do-i-get-groupberkeleyedu-a-hrefdocsservicesvhostvirtual-hostinga`